src/assets/projectJSS/Colors.ts

```diff
@@ -0,0 +1,16 @@
+// File contains all theme colors through the project, documented in 1st page of PowerPoint/Pdf
+
+// Usage: color: `${NAV_BAR_COLOR}`,
+
+export const NAV_BAR_COLOR = 'rgb(40,54,67)';

```

Comment:
I think thats what `theme` is for.   
You make global attributes that are provided by `materia-ui` to all the components.

Reply:
Yes, but the `theme` is not global for `all cases` (Not all buttons in project are green).

I left the all colors in 1 place for comfortable. If such color does repeat... then it is added inside the global theme 😃